### PR TITLE
fix(ls-hack): Supports dynamic PT for VP8.

### PR DESCRIPTION
**Pull request by gpolitis**
Chrome 57 changed the default payload types (e.g. previously it was 100 for VP8 and now it's 96). We're still using the old payload types and now Chrome seems to not register the associated payload type for VP8 RTX which breaks retransmissions. The workaround is to use the new payload type defaults that Chrome 57 is using, so the LS hack needs to dynamically discover the VP8 payload type and not always assume it's 100.

### CODE REVIEW

File: src/main/java/org/jitsi/videobridge/LipSyncHack.java [hunk 1]
```diff
@@ -318,7 +325,31 @@ void onRTPTranslatorWillWriteAudio(
      * @param sz the number of black key frame packets to create.
      * @return the array of black key frame packets that were created.
      */
-    private static RawPacket[] make(int ssrc, int seqNumOff, long tsOff, int sz)
+    private RawPacket[] make(int ssrc, int seqNumOff, long tsOff, int sz)
```
Comment (bbaldino) [inline src/main/java/org/jitsi/videobridge/LipSyncHack.java]: is it not possible/practical to pull these from signaling instead?  not that this wouldn't work, but technically we should be following what has been signaled, right?  might also make things more straightforward if we actively don't accept media for payload types that haven't been signaled.

Comment (gpolitis) [inline src/main/java/org/jitsi/videobridge/LipSyncHack.java]: This is exactly what this patch does ;)

Comment (bbaldino) [inline src/main/java/org/jitsi/videobridge/LipSyncHack.java]: lol, sorry!  when i saw it pulling from the stream i just assumed it was reading from an incoming packet.  just ignore me then :)

**Comment by bbaldino** [comment]
lgtm, will merge when tests finish

**Pull request merged**